statistics.test.py

```diff
@@ -12,11 +13,48 @@ def test_report_min_max_avg(self):
 
   def test_avg_is_nan_for_empty_input(self):
     computedStats = statistics.calculateStats([])
+    self.assertTrue(math.isnan(float(computedStats["avg"])))
+    self.assertTrue(math.isnan(float(computedStats["max"])))
+    self.assertTrue(math.isnan(float(computedStats["min"])))
     # All fields of computedStats (average, max, min) must be
     # nan (not-a-number), as defined in the math package
     # Specify the assert here.
     # Use nan and isnan in https://docs.python.org/3/library/math.html
 
+  def test_report_for_all_non_numeric_input(self):
+    computedStats = statistics.calculateStats(["1.5","d","abc","no","e"])
```

Comment:
Nice expression of the specification "if any number is invalid, the result is NaN"
Good habit to keep thinking "what else can go wrong?" 👍